findoutlie/metrics.py

```diff
@@ -22,11 +17,17 @@ def dvars(img):
         volumes in `img`.
     """
 
-    # Hint: remember 'axis='.  For example:
-    # In [2]: arr = np.array([[2, 3, 4], [5, 6, 7]])
-    # In [3]: np.mean(arr, axis=1)
-    # Out[2]: array([3., 6.])
-    #
-    # You may be be able to solve this in four lines, without a loop.
-    # But solve it any way you can.
+    try:
```

Comment:
This is generally a bad idea - to have a long `try` block.   In general prefer to let the error happen, and let the user interpret it.  But in any case - if you do want to do these kind of error checks - make them as short as possible - so you only catch the things you intended to catch, and so someone reading your code can see what you're trying to catch.

Reply:
Dear Matt,

  Thanks for your review and I understand your concept. And can you please review now. It will be benefit for me..